student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative('./person')
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(*args, classroom, **kwargs)
+    super(*args, **kwargs)
+    @classroom = classroom
+  end
```

Comment:
Great job @sonyco-4u 🤝

- [ ] You may make sure your constructor looks something like the snippet below. It is also good to have descriptive variable names.
![constructor](https://user-images.githubusercontent.com/63917725/210643913-7959ef0a-d24d-4ac6-94ac-e395b8b28290.PNG)